scoring/tests/test_end_state_scoring.py

```diff
@@ -223,6 +223,30 @@ def test_unclaimed_territory_with_others_claimed(self):
             },
         ])
 
+    def test_bronze_claim(self) -> None:
```

Comment:
Would be great to have a test case which includes one of the extra-points territories as well as a normal-points territory. That would validate that the points are being computed correctly per territory, rather than just having the overall points-per-territory value be modified.

Similarly it would be great to have a test for a combination of the different value extra-points territories.

Reply:
558a5cae85ff568cd20818e24ee7554a45f8a808